cmd/spire-server/cli/run/run.go

```diff
@@ -403,6 +405,7 @@ func NewServerConfig(c *Config, logOptions []log.Option) (*server.Config, error)
 		}
 		sc.Federation.FederatesWith = federatesWith
 	}
+	sc.Experimental.EnableAPI = c.Server.Experimental.EnableAPI
```

Comment:
We may want to log that the experimental Server API is enabled?

Reply:
:+1: